Automattic/jetpack
# Widgets: Remove Google+ Badge

See #10950.

#### Changes proposed in this Pull Request:
* Removes the Google+ Badge widget completely.
* Purges those widgets from all sidebars.

#### Testing instructions:
1. `git checkout 6.9`
2. Add a Google+ Widget to your sidebar. See it in your sidebar.
3. `wp option get widget_googleplus-badge`. See the widget data.
4. `git checkout remove/google-plus-widget`.
5. Reload the front page. See no Google+ Widget in your sidebar.
6. `wp option get widget_googleplus-badge`. See the option has been deleted.

#### Proposed changelog entry for your changes:
Removed Google+ Badge Widget.
